Find a functionally equivalent code:android.content.res.BridgeTypedArray.getTextArray:COMMENT
Method Modifier: public      
Comment:/**
 * Retrieve the CharSequence[] for the attribute at <var>index</var>.
 * This gets the resource ID of the selected attribute, and uses
 * {@link Resources#getTextArray Resources.getTextArray} of the owning
 * Resources object to retrieve its String[].
 *
 * @param index Index of attribute to retrieve.
 *
 * @return CharSequence[] for the attribute, or null if not defined.
 */

@@ -1,19 +1,7 @@
 {
-    if (index < 0 || index >= mResourceData.length) {
-        return null;
-    }
-    if (mResourceData[index] == null) {
-        return null;
-    }
-    String value = mResourceData[index].getValue();
+    String value = getString(index);
     if (value != null) {
-        if (RenderResources.REFERENCE_NULL.equals(value)) {
-            return null;
-        }
         return new CharSequence[] { value };
     }
-    Bridge.getLog().warning(LayoutLog.TAG_RESOURCES_FORMAT, String.format(// DEBUG
-    String.format(// DEBUG
-    "Unknown value for getTextArray(%d) => %s", index, mResourceData[index].getName())), null);
     return null;
 }

